docs/docs-beta/docs/integrations/dbt.md

```diff
@@ -34,47 +34,7 @@ pip install dagster-dbt
 
 ### Example
 
-```python
-from pathlib import Path
-
-from dagster import AssetExecutionContext, Definitions
-from dagster_dbt import (
-    DbtCliResource,
-    DbtProject,
-    build_schedule_from_dbt_selection,
-    dbt_assets,
-)
-
-RELATIVE_PATH_TO_MY_DBT_PROJECT = "./my_dbt_project"
-
-my_project = DbtProject(
-    project_dir=Path(__file__)
-    .joinpath("..", RELATIVE_PATH_TO_MY_DBT_PROJECT)
-    .resolve(),
-)
-my_project.prepare_if_dev()
-
-
-@dbt_assets(manifest=my_project.manifest_path)
-def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
-    yield from dbt.cli(["build"], context=context).stream()
-
-
-my_schedule = build_schedule_from_dbt_selection(
-    [my_dbt_assets],
-    job_name="materialize_dbt_models",
-    cron_schedule="0 0 * * *",
-    dbt_select="fqn:*",
-)
-
-defs = Definitions(
-    assets=[my_dbt_assets],
-    schedules=[my_schedule],
-    resources={
-        "dbt": DbtCliResource(project_dir=my_project),
-    },
-)
-```
```

Comment:
I can't comment directly on the line since it's not in the diff, but this is for line 45 in the new diff:

> Are you looking to learn more on running Dagster with dbt? Explore the <a href="https://courses.dagster.io/courses/dagster-dbt">Dagster University dbt course</a>.

Alerts keep going off for this - are you able to fix this while you're in here, or on a follow up PR? It can just be plain Markdown.